proxy/Main.cc

```diff
@@ -1771,10 +1777,13 @@ main(int /* argc ATS_UNUSED */, const char **argv)
   // Do the inits for NetProcessors that use ET_NET threads. MUST be before starting those threads.
   netProcessor.init();
   init_HttpProxyServer();
+  naVecMutex = new_ProxyMutex();
+  startAcceptMutex = new_ProxyMutex();
 
   // !! ET_NET threads start here !!
   // This means any spawn scheduling must be done before this point.
   eventProcessor.start(num_of_net_threads, stacksize);
+  total_net_threads = num_of_net_threads;
```

Comment:
Why have both `total_net_threads` and `num_of_net_threads`?

Reply:
didn't want to change the scope of that variable, but I guess there's no harm doing that